compiler/src/main/scala/org/scalajs/nscplugin/GenJSExports.scala

```diff
@@ -150,10 +150,8 @@ trait GenJSExports[G <: Global with Singleton] extends SubComponent {
             js.TopLevelJSClassExportDef(info.moduleID, info.jsName)
 
           case Constructor | Method =>
-            val exported = tups.map(t => ExportedSymbol(t._2))
-
             val methodDef = withNewLocalNameScope {
-              genExportMethod(exported, JSName.Literal(info.jsName), static = true)
+              genExportMethod(tups.unzip._2, JSName.Literal(info.jsName), static = true)
```

Comment:
```suggestion
              genExportMethod(tups.map(_._2), JSName.Literal(info.jsName), static = true)
```
I missed this one before.

(I'm pretty sure avoiding the creation of the list of `_1` is worth the creation of the lambda for `_._2`.)